NuGet.config

```diff
@@ -1,18 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-	<config>
-		<add key="repositorypath" value="packages" />
-	</config>
-	<packageSources>
-		<clear />
-		<add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" protocolVersion="3" />
-		<add key="Dotnet arcade" value ="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
-		<add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
-		<add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
-		<add key="xamarin-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/xamarin-impl/nuget/v3/index.json" />
-	</packageSources>
-	<disabledPackageSources>
-		<clear />
-	</disabledPackageSources>
+  <config>
```

Comment:
The changes in this file looks like whitespace only changes?

Reply:
The `darc` tool formatted this file as part of dependency updating.  I figure it's best to include those formatting changes in this initial PR so future auto PRs will only change version information.

@rolfbjarne one other related issue that I wanted your opinion on is that the update tooling currently requires that a `global.json` file exist in the root of the directory. I am going to test this later today, but I think we may need to change the way the `global.json` is generated as part of this PR as well. Do you have any initial thoughts on a preferred way to include a default `global.json`? It can be empty, it just needs to exist for the tool to run.

Reply:
If need a `global.json` in the repo, we have to stop generating it (or modifying it during the build), otherwise things will become annoying quite fast (with the file always showing up as modified).

I think the best option would be to:

* Stop generating it
* Commit the current generated result
* Compute DOTNET_VERSION from it
* Modify system-dependencies.sh to grep global.json instead of Make.config for the .NET version.

Reply:
Question, an we just do the trick I do with make + sed to stop needed to parse? Call make system-dependecies.sh and take the values from make.config.. I know, it add an extra step. Or maybe, is there a way to import the env values from the makefile?

Reply:
A root global.json file has been added, and the new `dotnet.config` make file now includes both `DOTNET_VERSION` and `DOTNET6_VERSION`. This keeps other `$(DOTNET_VERSION)` references working. `DOTNET_URL` has been moved from `Make.config` to `system-dependencies.sh`, and made "evergreen" by using a URL that stays constant (aside from the version).